<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EDFdialog</class>
 <widget class="QDialog" name="EDFdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EDF - Properties</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="subjectCodeInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SexInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="patientAdditionalInfoInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Additional info:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subjectNameInput">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Birthdate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Additional info:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="technicianInput">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="adminCodeInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="recordingAdditionalInfoInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Technician:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deviceInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthdateInput">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>441</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Local patient identification</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>441</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Local recording identification</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Simulation info</string>
   </property>
   <widget class="QLineEdit" name="durationInput">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="simulationStartInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration:</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>buttonBox</zorder>
  <zorder>subjectCodeInput</zorder>
  <zorder>label</zorder>
  <zorder>SexInput</zorder>
  <zorder>label_2</zorder>
  <zorder>patientAdditionalInfoInput</zorder>
  <zorder>label_3</zorder>
  <zorder>subjectNameInput</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>technicianInput</zorder>
  <zorder>adminCodeInput</zorder>
  <zorder>label_6</zorder>
  <zorder>recordingAdditionalInfoInput</zorder>
  <zorder>label_12</zorder>
  <zorder>deviceInput</zorder>
  <zorder>birthdateInput</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EDFdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EDFdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
